### PR TITLE
fix: fix support vue3

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#7

### CODE REVIEW

File: packages/vue-grid/src/Grid.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    let ref = null;
+
+    if (props.ref) {
+      ref = {
+        i: instance.$,
+        r: props.ref,
+      };
+    }
+    return {
+      shapeFlag: 17,
```
Comment (WoodNeck) [inline packages/vue-grid/src/Grid.ts:17]: How about using [ShapeFlags](https://github.com/vuejs/vue-next/blob/master/packages/shared/src/shapeFlags.ts) from `@vue/shared` instead?

```ts
import { ShapeFlags } from "@vue/shared";

const shapeFlag = ShapeFlags.ARRAY_CHILDREN | ShapeFlags.ELEMENT;
```

Comment (daybrush) [reply packages/vue-grid/src/Grid.ts:17]: @WoodNeck 

ShapeFlags exists only as a type in shared.

Comment (WoodNeck) [reply packages/vue-grid/src/Grid.ts:17]: Did you tested that?

https://github.com/vuejs/vue-next/blob/b46d8578fd7e0cb7f82620d21f2a0ad1094c34af/packages/runtime-core/src/componentRenderUtils.ts#L62

They're using it as a value in their code.

Comment (daybrush) [reply packages/vue-grid/src/Grid.ts:17]: I also checked the code. There are no ShapeFlags in the transformed code.

The values of ShapeFlags are also transformed.

Comment (WoodNeck) [reply packages/vue-grid/src/Grid.ts:17]: That's weird, but okay then.

As it's a magic number, please change that whenever possible.
Or, at least leave a comment with a link to https://github.com/vuejs/vue-next/blob/master/packages/shared/src/shapeFlags.ts there

Comment (daybrush) [reply packages/vue-grid/src/Grid.ts:17]: @WoodNeck 

Okay!

**Review by mixed — APPROVED** [review]
LGTM

**Review by WoodNeck — APPROVED** [review]

**Pull request merged**